new/LIT7032Trinity.xml

```diff
@@ -0,0 +1,69 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7032Trinity" xml:lang="en" type="work">
```

Comment:
Just so I understand, the pull request is for creating LIT7032, but the name of the pull request is LIT7031, correct?

Reply:
LIT7032Trinity is correct. I do not remember, why I opened it as LIT7031. It was probably a mistake by myself.